P5/Source/Specs/teidata.version.xml

```diff
@@ -9,21 +9,20 @@ See the file COPYING.txt for details
           ident="teidata.version">
   <desc versionDate="2013-11-20"
          xml:lang="en">defines the range of attribute values which may be used to
-  specify a TEI or Unicode version number.</desc>
+    specify a version number following the Semantic Versioning Specification.</desc>
   <desc xml:lang="fr"
          versionDate="2007-06-12">définit la gamme des valeurs d'attribut
   exprimant un numéro de version TEI.</desc>
   <content>
-      <dataRef name="token" restriction="[\d]+(\.[\d]+){0,2}"/>
+    <dataRef name="token" restriction="(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-((0|[1-9]\d*|\d*[\-a-zA-Z][\-0-9a-zA-Z]*)(\.(0|[1-9]\d*|\d*[\-a-zA-Z][\-0-9a-zA-Z]*))*))?(\+([\-0-9a-zA-Z]+(\.[\-0-9a-zA-Z]+)*))?"/>
```

Comment:
I have not hand-parsed the regular expression and seen that it makes sense. I did test it against all 71 test cases, and it correctly flags the first 31 as valid and the remaining 40 as invalid. So this looks like the correct W3C regular expression for SemVer.
So hurrah! to @peterstadler for a job well done. I think he should submit this to the SemVer folks, but hope he’ll coordinate with me, as I may have an entirely different regexp for them, as well.
That said, I think we should be much more restrictive for `@version` of `<TEI>` and `<teiCorpus>`. Something more akin to `4\.1(\.(0|[1-9]\d*))?(-(alpha|beta))?(\+([\-0-9a-zA-Z]+(\.[\-0-9a-zA-Z]+)*))?`.